troubleshooting/troubleshooting-push-notifications-are-currently-initializing.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Push notifications are currently initializing
+description: Push notifications are currently initializing error recieved when trying to register a device for push notifications on Android or iOS.
+type: troubleshooting
+page_title: Push notifications are currently initializing
+slug: troubleshooting-push-notifications-are-currently-initializing
+tags: push-notifications
+teampulseid:
+ticketid: 1073013, 907763
+pitsid:
+
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Notifications (Backend Services)</td>
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Push Notifications</td>
+  </tr>
+</table>
+
+## Description
+
+Trying to register the device for push notifications using *everlive.push.register()* returns an error message: "Push notifications are currently initializing".
+
+## Error Message
+"Push notifications are currently initializing"
+
+## Cause\Possible Cause(s)
+This error is returned from Telerik Platform JavaScript SDK and means that the initial call to *everlive.push.register()* method has not yet finished, but it was called again by the app.
+
+## Solution
+There could be several reasons why the initial call to *everlive.push.register()* method has not yet finished before being invoked again:

```

Comment:
Adding a loading spinner is not an actual solution because the application logic for registering for push notifications is usually not UI - bound and should not be.